apps/storybook/src/overlays/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,297 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import {
+  Autocomplete,
+  AutocompleteContent,
+  AutocompleteEmpty,
+  AutocompleteInput,
+  AutocompleteItem,
+  AutocompleteLabel,
+  AutocompleteTrigger,
+} from "@optiaxiom/react/unstable";
+import { useState } from "react";
+
+type Story = StoryObj<typeof Autocomplete>;
+
+export default {
+  component: Autocomplete,
+} as Meta<typeof Autocomplete>;
+
+type Book = {
+  author: string;
+  disabled: boolean;
+  id: string;
+  title: string;
+};
+
+const books: Book[] = [
+  {
+    author: "Harper Lee",
+    disabled: false,
+    id: "book-1",
+    title: "To Kill a Mockingbird",
+  },
+  {
+    author: "Lev Tolstoy",
+    disabled: false,
+    id: "book-2",
+    title: "War and Peace",
+  },
+  {
+    author: "Fyodor Dostoyevsky",
+    disabled: false,
+    id: "book-3",
+    title: "The Idiot",
+  },
+  {
+    author: "Oscar Wilde",
+    disabled: true,
+    id: "book-4",
+    title: "A Picture of Dorian Gray",
+  },
+  { author: "George Orwell", disabled: false, id: "book-5", title: "1984" },
+  {
+    author: "Jane Austen",
+    disabled: true,
+    id: "book-6",
+    title: "Pride and Prejudice",
+  },
+  {
+    author: "Marcus Aurelius",
+    disabled: false,
+    id: "book-7",
+    title: "Meditations",
+  },
+  {
+    author: "Fyodor Dostoevsky",
+    disabled: true,
+    id: "book-8",
+    title: "The Brothers Karamazov",
+  },
+  {
+    author: "Lev Tolstoy",
+    disabled: false,
+    id: "book-9",
+    title: "Anna Karenina",
+  },
+  {
+    author: "Fyodor Dostoevsky",
+    disabled: false,
+    id: "book-10",
+    title: "Crime and Punishment",
+  },
+];
+
+const languages = [
+  "English",
+  "French",
+  "German",
+  "Spanish",
+  "Portuguese",
+  "Russian",
+  "Japanese",
+  "Korean",
+  "Chinese (Simplified)",
+  "Italian",
+  "Dutch",
+  "Polish",
+  "Swedish",
+  "Danish",
+  "Finnish",
+  "Norwegian",
+  "Turkish",
+  "Arabic",
+  "Hindi",
+  "Bengali",
+  "Vietnamese",
+  "Thai",
+  "Indonesian",
+  "Greek",
+  "Czech",
+  "Romanian",
+  "Hungarian",
+  "Hebrew",
+  "Ukrainian",
+  "Swahili",
+  "Malay",
+  "Tagalog",
+  "Persian",
+  "Tamil",
+  "Urdu",
+  "Afrikaans",
+  "Bulgarian",
+  "Catalan",
+  "Croatian",
+  "Lithuanian",
+];
+
+const itemToString = (book: unknown) => {
+  return book ? (book as Book).title : "";
+};
+
+const itemToKey = (book: unknown) => {
+  return (book as Book).id;
+};
+
+const isItemDisabled = (book: unknown) => {
+  return (book as Book).disabled;
+};
+
+export const Basic: Story = {
+  render: function Basic() {
```

Comment:
let's accept args here
```suggestion
  render: function Basic(args) {
```

and forward it to the `Autocomplete` component

then in your `Disabled` story you can just change it to something like:

```ts
export const Disabled: Story = {
  ...Basic,
  args: {
    disabled: true,
  },
};
```

Reply:
actually now that i think about it - the `disabled` prop should go on the `AutocompleteInput` element - not on the `Autocomplete` element